src/coreclr/jit/compiler.cpp

```diff
@@ -3236,10 +3238,7 @@ void Compiler::compInitOptions(JitFlags* jitFlags)
     }
 #endif // FEATURE_CFI_SUPPORT
 
-#ifdef TARGET_LOONGARCH64
-    // Hot/cold splitting is not being tested on LoongArch64.
     opts.compProcedureSplitting = false;
```

Comment:
This seems to be overriding the value set above.

Reply:
Oops. Sorry!